docs/digger/installation/rhmap-properties.asciidoc

```diff
@@ -6,55 +6,56 @@ When using Digger from RHMAP, you can configure Digger by setting RHMAP properti
 |===
 | Property name | Environment variable | Description
 
-|buildfarm.version
```

Comment:
yeah i think it would read better for the user there if we use the variable syntax that we use in the installer. e.g. the same name we [use to set the var](https://github.com/fheng/rhmap-ansible/blob/master/roles/preflight_checks/vars/buildfarm.yml#L25) in the installer